mrmustard/lab/circuit.py

```diff
@@ -96,16 +100,13 @@ def is_unitary(self):
     def __len__(self):
         return len(self._ops)
 
-    def _repr_markdown_(self) -> str:
-        """Markdown string to display the object on ipython notebooks."""
-        header = f"#### Circuit  -  {len(self._ops)} ops  -  compiled = `{self._compiled}`\n\n"
-        ops_repr = [op._repr_markdown_() for op in self._ops]  # pylint: disable=protected-access
-        return header + "\n".join(ops_repr)
+    _repr_markdown_ = None
 
     def __repr__(self) -> str:
         """String to display the object on the command line."""
-        ops_repr = [repr(op) for op in self._ops]
-        return " >> ".join(ops_repr)
```

Comment:
We had this to be the string representation of a circuit as per @elib20 request. To make sure we are not breaking anything, let's ask him if he is still using this feature.

Reply:
we can use __str__ for that